src/app/onion/learning-object-builder/components/standard-outcomes/standard-outcomes.component.ts

```diff
@@ -110,6 +110,7 @@ export class StandardOutcomesComponent implements OnChanges, OnDestroy {
   }
 
   toggleStandardOutcome(standardOutcome: StandardOutcome, value: boolean) {
+    standardOutcome = new StandardOutcome(standardOutcome);
```

Comment:
What is happening here? Why are you creating a standardOutcome from the standardOutcome that was passed to the toggle function?

Reply:
The StandardOutcome is not typed correctly in parameter. It is just an object. There appears to be a larger issue with the StandardOutcome type in the client, but it will take some time to figure out.